src/routes/api/users.js

```diff
@@ -26,4 +22,5 @@ router.post("/user", (req, res, next) => {
   });
 });
 
+router.post("/auth/siginIn",validUser,login);
```

Comment:
Wouldn't it be better if the login API had the same format as other user-related APIs i.e adding `/user/` in the login API